lib/PluginFindRecord/PluginFindRecordModal.js

```diff
@@ -0,0 +1,401 @@
+                            >
+                              <FormattedMessage id="stripes-smart-components.search" />
+                            </Button>
+                          </div>
+                          <div className={css.resetButtonWrap}>
+                            <Button
+                              buttonStyle="none"
+                              disabled={disableReset()}
+                              fullWidth
+                              id="clickable-reset-all"
+                              onClick={resetAll}
+                            >
+                              <Icon icon="times-circle-solid">
+                                <FormattedMessage id="stripes-smart-components.resetAll" />
+                              </Icon>
+                            </Button>
+                          </div>
+                          {renderFilters
+                            ? renderFilters(activeFilters.state, getFilterHandlers())
+                            : (
+                              <Filters
+                                activeFilters={activeFilters}
+                                config={filterConfig}
+                                onChangeHandlers={getFilterHandlers()}
+                              />
+                            )
+                          }
+                        </form>
+                      </Pane>
+                    }
+                    <Pane
+                      firstMenu={this.renderResultsFirstMenu(activeFilters)}
+                      paneTitle={RESULTS_HEADER}
+                      paneSub={resultPaneSub}
+                      defaultWidth="fill"
+                      padContent={false}
+                    >
+                      <MultiColumnList
+                        visibleColumns={builtVisibleColumns}
+                        contentData={records}
+                        totalCount={count}
+                        id="list-plugin-find-records"
+                        columnMapping={{
+                          isChecked: (
+                            <Checkbox
+                              checked={isAllChecked}
+                              data-test-find-records-modal-select-all
+                              onChange={this.toggleAll}
+                              type="checkbox"
+                            />
+                          ),
+                          ...columnMapping,
+
+                        }}
+                        formatter={mixedResultsFormatter}
+                        onRowClick={!isMultiSelect && onSelectRow}
+                        onNeedMoreData={onNeedMoreData}
+                        onHeaderClick={onSort}
+                        sortOrder={sortOrder.replace(/^-/, '').replace(/,.*/, '')}
+                        sortDirection={sortOrder.startsWith('-') ? 'descending' : 'ascending'}
+                        isEmptyMessage={resultsStatusMessage}
+                        autosize
```

Comment:
Can we alpha-sort all these props? Ideally throughout the PR but esp in these long prop lists.

Reply:
I think such kind of things should be defined in a project style-guide, eslint, prettier , etc. In other cases it's per dev choice, and in Acquisitions we don't have such rule. However, I'll change it here, despite the fact that stripes-smart-components doesn't have this rule defined in eslint config.